app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+import Button, { Appearance, ButtonGroup } from '@atlaskit/button';
+import { ConnectedState } from '../StatusLabel/StatusLabel';
+import {
+	connectionPanelContainerContainer,
+	connectionPanelContainerHeader,
+	connectionPanelContainerParagraph
+} from './ConnectionPanel.styles';
+import { ConnectionPendingIcon } from '../icons/ConnectionPendingIcon';
+import { NoDataIcon } from '../icons/NoDataIcon';
+import { DuplicateServerIcon } from '../icons/DuplicateServerIcon';
+
+type NotConnectedStateProps = {
+	connectedState: ConnectedState;
+	contentHeader: string,
+	contentInstructionOne: string,
+	contentInstructionTwo?: string,
+	buttonAppearance: Appearance,
+	firstButtonLabel: string,
+	secondButtonLabel?: string,
+	buttonOneOnClick(data?: any): void,
+	buttonTwoOnClick?(): void,
+	testId?: string
+};
+
+const ConnectionPanelContent = ({
+	connectedState,
+	contentHeader,
+	contentInstructionOne,
+	contentInstructionTwo,
+	buttonAppearance,
+	firstButtonLabel,
+	secondButtonLabel,
+	buttonOneOnClick,
+	buttonTwoOnClick,
+	testId
+}: NotConnectedStateProps): JSX.Element => {
+	let icon;
+
+	if (connectedState === ConnectedState.CONNECTED) {
+		icon = <NoDataIcon />;
+	} else if (connectedState === ConnectedState.PENDING) {
+		icon = <ConnectionPendingIcon />;
+	} else {
+		icon = <DuplicateServerIcon />;
+	}
+
+	return (
+		<div className={cx(connectionPanelContainerContainer)}>
```

Comment:
This is basically what I had in NotConnectedState before but the design and content updates now include the same layout for CONNECTED servers when there is no pipeline data. Made sense to pull it out into a reusable component.